www/importentity.php

```diff
@@ -228,6 +230,7 @@
 
 function janus_array_diff_recursive($array1, $array2)
 {
+    if (!is_array($array2)) return $array1;
```

Comment:
Yes, I have troubles getting pull request clean from all other fixes I came up with. I thought I'd managed to keep it out, but apparently I didn't.